### PR TITLE
Expanse template updated

### PR DESCRIPTION
**Pull request by chrisfranko**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@chrisfranko__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: go-expanse-on-ubuntu/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Expanse installed
+
+2. A script to activate an Expanse node and begin interacting with the Expanse protocol.
+
+![Expanse-Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/go-expanse-on-ubuntu/images/exp.png)
+
+# Template Parameters
+When you launch the installation of the cluster, you need to specify the following parameters:
+* `newStorageAccountNamePrefix`: make sure this is a unique identifier. Azure Storage's accounts are global so make sure you use a prefix that is unique to your account otherwise there is a good change it will clash with names already in use.
```
Comment (singhkay) [inline go-expanse-on-ubuntu/README.md]: Should be removed since it isn't there in the template parameters

File: go-expanse-on-ubuntu/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+* `vmDnsPrefix`: this is the public DNS name for the VM that you will use interact with your gexp console. You just need to specify an unique name.
```
Comment (singhkay) [inline go-expanse-on-ubuntu/README.md]: This is different in the template as well

**Comment by chrisfranko** [comment]
updated, hopefully everything is good now,

**Comment by singhkay** [comment]
@chrisfranko When trying to deploy this VM fails with the error 

`Script returned an error.\n---stdout---\n\n---errout---\nsh: 0: Can't open configure-geth.sh` 

Can you check?

**Comment by chrisfranko** [comment]
:)

File: go-expanse-on-ubuntu/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Gexp Private Node
+
+This Microsoft Azure template deploys a single Ethereum client with a private chain for development and testing.
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fgo-ethereum-on-ubuntu%2Fazuredeploy.json)
```
Comment (singhkay) [inline go-expanse-on-ubuntu/README.md]: Link needs to point to this template

**Comment by singhkay** [comment]
@chrisfranko Found one small fix, we should be good after that

**Comment by chrisfranko** [comment]
@singhkay good eye man, this should be it.

**Comment by singhkay** [comment]
cancelling CI since it already passed, lgtm. merged

**Comment by chrisfranko** [comment]
celebrate!

**Comment by singhkay** [comment]
:beers:

**Comment by chrisfranko** [comment]
Thank you for all your help. :beer:

**Pull request merged**